Hello/app.ts

```diff
@@ -0,0 +1,5 @@
+window.onload = function(){ 
+    	$("#but").on("click",function(){
```

Comment:
You can do something like:
`<div id="helloButton" onclick="showHelloMessage()">Hello</div>`

And in the typescript, you create a function:
```
function showHelloMessage() {
  let textLabel = document.getElementById('hello');
  textLabel.innerText = "Your message";
}
```